frontend/src/components/MapView/DateSelector/index.tsx

```diff
@@ -97,6 +103,98 @@ const DateSelector = memo(() => {
     [availableDates],
   );
 
+  const AAAvailableDates = useSelector(AAAvailableDatesSelector);
```

Comment:
Moving these up from TimelineItems to make use of the processed layers that are created